java/spark-bundle/build.gradle.kts

```diff
@@ -14,8 +14,8 @@ plugins {
     id("com.github.johnrengelman.shadow") version ("5.2.0")
 }
 
-val scalaVersion = project.properties.getOrDefault("scalaVersion", "2.12")
-val sparkVersion = project.properties.getOrDefault("sparkVersion", "3.1.1") as String
```

Comment:
We should at least keep publishing the sparkVersion we were: 3.1.1, looks like your change switches us to 2.4.7 only? Can you update that?